docker-hub/builds/index.md

```diff
@@ -17,6 +17,10 @@ Docker Hub can automatically build images from source code in an external
 repository and automatically push the built image to your Docker
 repositories.
 
+> **Note**
+>
+> The Autobuilds feature is available for [Pro](https://hub.docker.com/billing/plan/update){: target="_blank" rel="noopener" class="_"} or [Team](https://hub.docker.com/subscription?plan=team&paid=true){: target="_blank" rel="noopener" class="_"} plan only.
```

Comment:
Suggest we point to the Docker Pricing page for details on Pro and Team subscription